surround/remote/local.py

```diff
@@ -17,6 +17,7 @@ def file_exists_on_remote(self, path_to_remote, relative_path_to_remote_file, ap
 
     def pull_file(self, what_to_pull, key, path_to_remote, relative_path_to_remote_file, path_to_local_file):
         path_to_remote_file = os.path.join(path_to_remote, relative_path_to_remote_file)
+        os.makedirs(os.path.dirname(path_to_local_file), exist_ok=True)
```

Comment:
I don't think this is required as it is already present in pull. Do you have a case where it fails without this?

Reply:
Yeah when I had a file in a sub-folder e.g. `testing/test.data` and ran `pull` in a version of the Surround project without the files pulled, it would fail because the `testing` folder does not exist